Kconfig

```diff
@@ -246,4 +246,12 @@ config USE_NODEJS
 	default n
 	help
 		Enable Node.js bindings
+
+config USE_PYTHON
+	bool "Python bindings"
+	depends on HAVE_LIBPYTHON3 && HAVE_CYTHON3 && HAVE_PYTHON3
+	default n
```

Comment:
sure. People can always disable if they do not want to.

it would be nice to get the semaphore-ci to build it, then it needs the deps installed.